returnn/dataset.py

```diff
@@ -0,0 +1,90 @@
+__all__ = ["ExtractDatasetStatisticsJob"]
+
+from sisyphus import *
+
+import numpy
```

Comment:
numpy is not an python buildin lib and thus should be in it's own block (after the python libs)